yew-macro/src/html_tree/html_tag/mod.rs

```diff
@@ -194,15 +194,44 @@ impl ToTokens for HtmlTag {
                      value,
                  }| {
                     let key = label.to_lit_str();
-                    if question_mark.is_some() {
-                        let sr = stringify::stringify_option_at_runtime(value);
-                        quote! {
-                            ::yew::virtual_dom::PositionalAttr(#key, #sr)
+                    match value {
+                        Expr::Paren(ExprParen { expr, .. }) if label.name == "class" => {
```

Comment:
if you remove `if label.name == "class"`, the `()` syntax can be used everywhere.

This would be valid for example:

```rust
<div style=("background-color: blue;", "color: green;") />
```

But this too:

```rust
<a href=() />
```
(translates to empty string)

It might be good. Idk. Let me know!